scripts/create_test_forms.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+'''Generate test form entries/items as JSON to use for load testing.
+'''
+
+import json
+import csv
+import uuid
```

Comment:
Nit: Do we care about alphabetizing these?

Reply:
@sharonwarner and I agreed for node/react that it makes more sense to order imports by logical grouping rather than alphabetically, so we disabled the default alphabetical imports linter rule. that said, in this file i might as well make them alphabetical! i'm submitting another PR now for this script so i'll do that there